ouroboros-consensus/src/Ouroboros/Consensus/Node/ExitPolicy.hs

```diff
@@ -0,0 +1,31 @@
+module Ouroboros.Consensus.Node.ExitPolicy
+  ( NodeToNodeClientResult (..)
+  , returnPolicy
+  -- Re-exports
+  , ReturnPolicy
+  ) where
+
+import           Ouroboros.Network.ExitPolicy
```

Comment:
`stylish-haskell` might also object to these not being in alphabetical order. If you want them in this order, you can add a newline between them.

Reply:
fixed